### ISSUE TITLE
 how to recognize the blank between two English words

### ISSUE THREAD

**Issue by cvchongci** [clovaai/deep-text-recognition-benchmark#161]
How to recognize blank between two English words?
for my current model, if I input one English sentence then the output will concatenate all the English words together. for example:
inputed image:
<img width="363" alt="36" src="https://user-images.githubusercontent.com/62248916/80300616-539c4c00-87af-11ea-93d8-1ba1b150594d.png">
recognized result:

> A------l-ll-t-h--e--r-e-c--o--g-n-i-tiio---n--a-c-cc--ur-a--c-i-e-s---o--n--t-h---e => Alltherecognitionaccuraciesonthe

So how to recognize the blank between two English words?

Thank you so much.

**Comment by YacobBY** [comment]
In the CRAFT project you need to set link_threshold and low_text higher so that it separates the words before you pass them as images to the OCR. CRAFT is responsible for cleanly separating text, this project just reads the images without knowing what a space is.

**Comment by cvchongci** [comment]
@YacobBY thanks for the reply.

**Comment by rexlow** [comment]
@YacobBY Yes, the CRAFT project does exactly that. But afterward, how do we connect back the words that belong in the same sentence?

**Comment by fingoldo** [comment]
> @YacobBY Yes, the CRAFT project does exactly that. But afterward, how do we connect back the words that belong in the same sentence?

probably by computing proximities of bounding boxes

**Comment by AhmadZobairSurosh** [comment]
@YacobBY 

Hi
Did you find a solution for this problem? I mean connecting back the words that belong in the same sentence. 
In my case its numbers separated by . 

Thanks in advance
